scripts/eks/appsignals/deploy-sample-app.sh

```diff
@@ -33,6 +33,13 @@ if [[ $OPERATION == "apply" ]]; then
         --attach-policy-arn arn:aws:iam::aws:policy/AmazonBedrockFullAccess \
         --approve \
         --override-existing-serviceaccounts
+
+    aws dynamodb create-table \
```

Comment:
Creating the dynamodb table here is strange here and error prone. What if the table is already exist? We usually let the sample app to create its own resources so that it's self contained. 

Can we do it inside the payment service for consistency? Like what the java services do, we want to create the table only if it doesn't exist.

Reply:
I added logic to check if the table exists before creating it.